### PR TITLE
Contributing CI docs tidying

### PR DESCRIPTION
**Pull request by hflynn**
Removes Figure from consortium page and tidies intro page to make it version neutral so we don't have to keep updating it.

N.B. merge job is still waiting on Roger updating the C++ jobs on the release builds page so isn't expected to be made green by this PR.

--no-rebase

### CODE REVIEW

File: contributing/ci-introduction.txt [hunk 1]
```diff
@@ -29,28 +29,23 @@ Most of the OME code is split between four repositories: openmicroscopy.git_,
...
-the name of the branch which they will be merged into.
-
-The 5.1.x series consists of PRs labeled with “dev_5_1”, which is also
-the name of the branch which they will be merged into.
-
-The 5.2.x series consists of PRs labeled with “develop”, which is also
-the name of the branch which they will be merged into.
+Each release series consists of PRs labeled according to the release version,
+which also matches the name of the branch they will be merged into e.g. 5.1.x
+series PRs will be labelled as "dev_5_1" and be merged into the dev_5_1
```
Comment (mtbc) [inline contributing/ci-introduction.txt]: maybe "labeled" to look less British

File: contributing/ci-consortium.txt [hunk 2]
```diff
@@ -93,33 +89,15 @@ https://github.com/openmicroscopy/Imperial-FLIMfit
...
 		   https://github.com/openmicroscopy/ome-release
-		#. Merge PRs opened against `dev_5_0`
+		#. Merge PRs opened against `develop`
 		#. Copy the FLIMfit Windows artifacts over SSH to
 		   :file:`/ome/apache_repo/public/flimfit/RELEASE`
 		#. Build the FLIMfit downloads pages using :command:`make flimfit`
 		#. Copy the FLIMfit downloads page over SSH to
 		   :file:`/ome/apache_repo/public/flimfit/RELEASE`
 
-OMERO.figure
```
Comment (sbesson) [inline contributing/ci-consortium.txt]: What was the rationale for removing this paragraph? I think we are still using these downloads page/job to create OMERO.figure downloads.

Comment (hflynn) [inline contributing/ci-consortium.txt]: Because it isn't a consortium job any more and this page is all about partner projects not mainline things.

We can put it on the release jobs page instead if you like, I've added the figure-help builds to the docs builds page already.

Comment (sbesson) [inline contributing/ci-consortium.txt]: Putting it under the release jobs page sounds good. Do you want to do that here or in a separate PR?

Comment (hflynn) [inline contributing/ci-consortium.txt]: I'll consult with Roger on that, didn't want to touch that page while he was prepping his C++ changes but I don't know if he's started yet or not.

Comment (hflynn) [inline contributing/ci-consortium.txt]: Roger is working on the page now so I'll do it on a new PR after his has gone in.

**Comment by sbesson** [comment]
:+1: Ready to merge once Travis is green
